src/engine/effects/engineeffectchain.h

```diff
@@ -64,9 +64,9 @@ class EngineEffectChain final : public EffectsRequestHandler {
     bool updateParameters(const EffectsRequest& message);
     bool addEffect(EngineEffect* pEffect, int iIndex);
     bool removeEffect(EngineEffect* pEffect, int iIndex);
-    bool enableForInputChannel(const ChannelHandle* inputHandle,
+    bool enableForInputChannel(const ChannelHandle inputHandle,
```

Comment:
Please remove the `const` qualifiers from all by-value arguments.

Reply:
I believe as long as we call memset on the largest struct item, that will be sufficient for zeroing out the data